package.json

```diff
@@ -18,6 +18,9 @@
     "nightwatch": "0.9.6",
     "nightwatch-commands": "1.7.0"
   },
+  "engines": {
+      "node": "^4.0.0"
```

Comment:
I tried running `npm install` using node v6.6.0 and it ran fine.  Do you think we should add an `.npmrc` file to the repo and use the [engine-strict](https://docs.npmjs.com/misc/config#engine-strict) config setting?